woocommerce/woocommerce-android
# Feature/10191 remove read only subscription code

Closes: #10191 


### Description

2 things in this PR: 
- Removes all code related to subscription `read-only` section
- Remove feature flag `PRODUCT_SUBSCRIPTIONS` completely

### Testing instructions

1. Log into a subscription eligible site
2. Select or create a subscription product
3. Check subscription read only section is gone
4. Smoke test subscription editing.
